propel.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+- name: reaction-core
+  certificate_arn: arn:aws:acm:us-west-2:773713188930:certificate/c2979a7a-7b84-43ed-b3a7-24b0256a1b9f
+  root_domain: staging.reactioncommerce.com
+  dns_name: reaction-core
+  desired_task_count: 2
+  min_task_count: 0
+  max_task_count: 2
+  min_healthy_percent: 100
+  max_percent: 200
+  alb_listener_port: 80
+  alb_listener_path: /
+  alb_health_check_path: /health
+  task-definition:
+    name: staging-core
+    containers:
+    - name: core
+      cpu: 500
+      memory: 1800
+      port_mappings:
+      - container_port: 3000
+        host_port: 3000
+      image: reactioncommerce/reaction
+      image_tag: release-2.0.0-rc.5
+      env_params:
+      - name: REACTION_AUTH
+      - name: REACTION_EMAIL
+      - name: REACTION_USER
+      - name: MONGO_URL
+      - name: ROOT_URL
+      - name: HYDRA_ADMIN_URL
+      - name: HYDRA_OAUTH2_INTROSPECT_URL
+      - name: SKIP_FIXTURES
```

Comment:
Does this list need to change when we add env variables? Where do the values come from? Maybe we can make this more straightforward when we do #4706

Reply:
Those are there for informational purposes only. I can remove them once we have the .env.example file.